spec/factories.rb

```diff
@@ -39,8 +41,14 @@
       state        'created'
     end
 
+    trait :single_claimant do
+      without_additional_claimants_csv
+      after(:create) { |claim| claim.secondary_claimants.clear }
```

Comment:
Does clear destroy the collection or null out foreign keys? Prob fine if the tests only only use instances returned by factory girl.